src/FSharp.Core/result.fsi

```diff
@@ -283,6 +283,21 @@ module Result =
     [<CompiledName("ToList")>]
     val toList: result: Result<'T, 'Error> -> 'T list
 
+    /// <summary>Convert the result to a sequence of count 0 or 1.</summary>
```

Comment:
Where did this comment come from, did you write it from scratch?

Reply:
Yes, I copied the `ToList` bits and winged it.